torch/_subclasses/meta_utils.py

```diff
@@ -779,10 +779,6 @@ def empty_create_subclass(
             # NB: t.ctx could be None if the subclass in question has no
             # meaningful context
 
-            assert symbolic_context is None or isinstance(
```

Comment:
can you explain why we don't need the assert anymore?

Reply:
oh nvm, code motion? (I see it lower down)

Reply:
Yep i changed this function to be recursive